birth-week-day-page/birth-week-day-page.html

```diff
@@ -12,11 +12,25 @@
 </head>
 <body class="d-flex min-vh-100 justify-content-center align-items-center">
 <main class="card text-start col-xl-4 col-lg-6 col-md-8 col-11">
-    <div class="card-header text-center py-3">
-        <h2>Birth week day calculator</h2>
+    <div class="card-header">
+        <ul class="nav nav-pills card-header-pills">
+            <li class="nav-item">
+                <a class="nav-link" href="../chat/chat.html">Chat</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link active" href="../birth-week-day-page/birth-week-day-page.html">Birthday</a>
```

Comment:
Please, change url to self page to more simple

Reply:
fixed